### DISCUSSION TITLE
Bolt 3.* - /home/domain,com/app/cache/profiler getting HUGE!

### DISCUSSION THREAD

**Discussion by 4044ever** [bolt/core#3345] [General]
My VPS server is running out of space. I have two domains, both are Bolt (3.* forgot which exactly and can't login anymore, might be that the admin folder got deleted).

Anyway, my VPS has 20Gb and `/home/domain,com/app/cache/profiler` is getting near 10Gb, VPS now at 100% space.

Can I delete what's in 'profiler' safely? 

And is there a way to login again? (seems /bolt/ got deleted)

**Comment by rossriley** [comment]
Yes you can delete it and most importantly don't have debug switched on in your config on production. This is the cause of it filling up.

**Reply by 4044ever** [reply]
Thank you for replying. Meanwhile I delete all the folder older than 2 days and server space is from 100% down to 54%

How I can get back to admin? seems silly me delete the /bolt/ folder
